### PR TITLE
Swapping out manual istio install with hosted istio

### PR DESCRIPTION
**Pull request by raddaoui**

### CODE REVIEW

**Comment by raddaoui** [comment]
ok to test

**Comment by raddaoui** [comment]
ok to test

File: scripts/teardown.sh [hunk 1]
```diff
@@ -15,14 +15,19 @@
...
-kubectl delete deploy,svc --all -n default
-kubectl delete deploy,svc --all -n istio-system
-kubectl delete deploy,svc --all -n vm
+gcloud beta container clusters update "$ISTIO_CLUSTER" \
+    --update-addons=Istio=DISABLED \
+    --project "$ISTIO_PROJECT" \
+    --zone "$ZONE"
+
+kubectl delete ns vm --ignore-not-found=true
+kubectl delete ns bookinfo --ignore-not-found=true
```
Comment (chrislovecnm) [inline scripts/teardown.sh]: We need to wait till this are cleaned up as well

Comment (chrislovecnm) [reply scripts/teardown.sh]: I am gonna merge but we need another PR in for us to wait on the services to delete

**Pull request merged**